src/components/settings/Settings.tsx

```diff
@@ -49,86 +47,102 @@ export function Settings() {
     }
   }, [project.data]);
 
+  const addCollaboratorHandler = useCallback(
+    () =>
+      modals.openContextModal({
+        modal: Modals.CollaboratorModal,
+        title: "Add Collaborator",
+        innerProps: {
+          handleSubmit: project.createCollaborator,
+        },
+      }),
+    [project.createCollaborator],
+  );
+
+  const deleteTestbookHandler = useCallback(
+    () =>
+      modals.openContextModal(
+        merge(deleteModalsDefaults, {
```

Comment:
Maybe export the "openContextModal" with the merge in a dedicated function for delete modals? 

```
const openDeleteConfirmModal = (title: string, innerProps: TConfirmModalProps) => modals.openContextModal(
        merge(deleteModalsDefaults, {
          title,
          innerProps
      })),
```